.github/workflows/ci.yml

```diff
@@ -8,6 +8,42 @@ on:
   pull_request:
 
 jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        id: toolchain
+        with:
+          toolchain: stable
+          profile: minimal
+          components: clippy, rustfmt
+          override: true
+
+      - run: make init-submodules
```

Comment:
do you rather want to use the `submodules: recursive` setting of the checkout action?
https://github.com/actions/checkout#usage

Reply:
Yeah, let's do that.